AutoGraph/AutoGraph.swift

```diff
@@ -101,23 +101,23 @@ open class AutoGraph {
     /// immediate failure, in which case, please check the error received in the completion block.
     ///
     /// It will form a websocket connection if one doesn't exist already.
-    open func subscribe<R: Request>(_ request: R, completion: @escaping RequestCompletion<R.SerializedObject>) -> Subscriber? {
+    open func subscribe<R: Request>(_ request: R, connectionState: @escaping SubscriptionResponseHandler.WebSocketConnectionBlock, completion: @escaping RequestCompletion<R.SerializedObject>) -> Subscriber? {
```

Comment:
@rex-remind101 So thinking of a way to update the UI of a connection state change to let the user know if the web socket is reconnecting. I thought adding another block would work